pkg/agent/openflow/client.go

```diff
@@ -114,6 +114,9 @@ type Client interface {
 	InstallServiceFlows(groupID binding.GroupIDType, svcIP net.IP, svcPort uint16, protocol binding.Protocol, affinityTimeout uint16) error
 	// UninstallServiceFlows removes flows installed by InstallServiceFlows.
 	UninstallServiceFlows(svcIP net.IP, svcPort uint16, protocol binding.Protocol) error
+	// InstallLoadBalancerServiceFromOutsideFlows installs flows for LoadBalancer Service traffic from outside.
+	// The traffic will be forwarded to kube-proxy by the installed flows.
+	InstallLoadBalancerServiceFromOutsideFlows(svcIP net.IP, svcPort uint16, protocol binding.Protocol) error
```

Comment:
I think this is for Windows only? Probably mention it in the comments, and also explain the traffic is from uplink port and forwarded to gateway port.

Reply:
Yes, the function is only for Windows. Add comments for it. Done.